_1327/user_management/tests.py

```diff
@@ -102,3 +106,38 @@ def test_impersonate_wrong_url(self):
 
 		response = self.app.post('/hijack/{username}/'.format(username=user.username), user=self.user, expect_errors=True)
 		self.assertEqual(response.status_code, 400)
+
+
+class _1327AuthenticationBackendTests(WebTest):
```

Comment:
Please have a Look at these tests again, as they do not really reflect the behaviour currently implemented in the authenticationbackend